pages/index.js

```diff
@@ -6,17 +6,20 @@ import SpeakersSection from '../components/SpeakersSection'
 import TicketsSection from '../components/TicketsSection'
 import AboutSection from '../components/AboutSection'
 import SponsorsSection from '../components/SponsorsSection'
+import LogoBackground from '../components/LogoBackground'
 
 export default () => (
   <MainLayout>
     <HeroUnit />
     <SectionSeparator first />
+    <LogoBackground width={820} top={100} right={-200} />
```

Comment:
Since `top` is relative to the current flowing position, I recommend renaming the `top` prop to `offsetTop`.